packages/wonder-blocks-dropdown/components/dropdown.test.js

```diff
@@ -257,5 +257,63 @@ describe("Dropdown", () => {
             // Assert
             expect(controlledComponent.state("selectedValues")).toBeFalsy();
         });
+
+        it("only single selects OptionItems if singleSelectOption is passed", () => {
+            // Arrange
+            const controlledComponent = mount(
+                <ControlledComponent singleSelectOption />,
+            );
+            controlledComponent.simulate("click");
+            controlledComponent.setState({selectedValues: ["B"]});
+            const dropdownCore = controlledComponent.find(DropdownCore);
+
+            // Act
+            dropdownCore.simulate("keydown", {keyCode: keyCodes.down});
+            dropdownCore.simulate("keyup", {keyCode: keyCodes.down});
+
+            const optionItem = controlledComponent.find(OptionItem).at(0);
+            optionItem.simulate("click");
+
+            // Assert
+            expect(controlledComponent.state("selectedValues")).toHaveLength(1);
```

Comment:
This indicates a slight inconsistency in the API between `SingleSelect` and `Dropdown` in select-select mode.  `SingleSelect`'s `onChange` passes back a single value where as `Dropdown`'s `onChange` passes an array even in single select mode.  I don't think this is something we need to solve in this PR.  Maybe this inconsistency is fine, but it is something to think about.